app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```

Comment:
use `"` instead of `'`

Reply:
I have resolved this for line 5 to 8. But may I know, other than line 5 to 8, do I need to use " instead ' for other lines too ?